server/functions/__tests__/mocks/api/index.ts

```diff
@@ -0,0 +1,64 @@
+import dotenv from 'dotenv';
+dotenv.config();
+import express from 'express';
+import cors from 'cors';
+
+const app = express();
+app.use(express.json());
+app.use(cors());
+
+app.get('/', (req, res) => {
+    res.status(200).send("it's alive!");
+});
+
+app.post('/client', (req, res) => {
+    const data = req.body;
+    if (!data.id) {
+        res.status(400).send({
+            message: 'Missing id field',
+        });
+    } else {
+        res.status(201).send({
+            message: 'Created',
+        });
+    }
+});
+
+app.get('/client/:clientId', (req, res) => {
+    res.status(200).send({
+        id: req.params.clientId,
+    });
+});
+
+app.post('/measurement', (req, res) => {
+    const data = req.body;
+    if (!data.clientID
+                || !data.data
+                || !data.data.date || !data.data.dataType || !data.data.value) {
+        res.status(400).send({
+            message: 'Missing data fields',
+        });
+    } else if (isNaN(data.data.date)) {
+        res.status(400).send({
+            message: 'date must be a number',
+        });
+    } else {
+        res.status(201).send({
+            message: 'Created',
+        });
+    }
+});
```

Comment:
would it be possible to add a basic state variable (key-value map is fine) and an endpoint to get the state? We can use that to test which data is sent to the API.

Reply:
Yep it's possible! Added as 
```
data: {
    uid: string,
    value: number,
    name: string,
    source: string,
    date: number,
}
```